testsuite/features/secondary/min_salt_migration.feature

```diff
@@ -0,0 +1,173 @@
+    Then I should see a "Remote Commands" text in the content area
+    When I enter command "file /etc/salt"
+    And I click on preview
+    Then I should see "sles-salt-migration.tf.local" hostname
+    And I wait until I do not see "pending" text
+    When I click on run
+    And I wait until I do not see "pending" text
+    And I expand the results for "sles-salt-migration.tf.local"
+    Then I should see "/etc/salt: cannot open `/etc/salt' (No such file or directory)" in the command output for "sles-salt-migration.tf.local"
+    When I enter command "file /etc/venv-salt-minion"
+    And I click on preview
+    Then I should see "sles-salt-migration.tf.local" hostname
+    And I wait until I do not see "pending" text
+    When I click on run
+    And I wait until I do not see "pending" text
+    And I expand the results for "sles-salt-migration.tf.local"
+    Then I should see "/etc/venv-salt-minion: directory" in the command output for "sles-salt-migration.tf.local"
+
+  Scenario: Do some basic testing on the SLES 15 KVM VM with the Salt bundle
+    Given I am on the Systems page
+    When I follow "sles-salt-migration"
+    And I follow "Software" in the content area
+    And I follow "Install"
+    And I enter "adobe-sourcecodepro-fonts" as the filtered package name
+    And I click on the filter button
+    And I wait until I see "adobe-sourcecodepro-fonts" text
+    And I check "adobe-sourcecodepro-fonts" in the list
+    And I click on "Install Selected Packages"
+    Then I click on "Confirm"
+    When I wait until event "Package Install/Upgrade scheduled by admin" is completed
+    Then "adobe-sourcecodepro-fonts" should be installed on "sles-salt-migration"
+    And I follow "Software" in the content area
+    And I follow "Install"
+    When I follow "List / Remove"
+    And I enter "adobe-sourcecodepro-fonts" as the filtered package name
+    And I click on the filter button
+    And I wait until I see "adobe-sourcecodepro-fonts" text
+    And I check "adobe-sourcecodepro-fonts" in the list
+    And I click on "Remove Packages"
+    Then I click on "Confirm"
+    When I wait until event "Package Install/Upgrade scheduled by admin" is completed
+    Then "adobe-sourcecodepro-fonts" should not be installed on "sles-salt-migration"
+
+  Scenario: Cleanup: delete created SLES 15 KVM VM in the Salt migration context
+    Given I am on the Systems page
+    When I follow "sles-salt-migration.tf.local"
+    And I follow "Delete System"
+    Then I should see a "Confirm System Profile Deletion" text
+    When I click on "Delete Profile"
+    And I wait until I see "has been deleted" text
+    Then "sles-salt-migration.tf.local" should not be registered
+    When I delete the virtual machine named "sles-salt-migration" on "kvm_server"
+    And I delete salt-sles virtual network on "kvm_server"
+    And I delete salt-sles virtual storage pool on "kvm_server"
+    And I delete tmp virtual storage pool on "kvm_server"
```

Comment:
Can we try to write this using API/Command-line steps?

Reply:
For deleting the VM/virtual network/virtual storage we already use the command line with `virsh`. I can have a look on how to delete the system.

Reply:
I found a solution for deleting a system using the `spacecmd` API.